pkg/dashboard/handlers/helmHandlers.go

```diff
@@ -183,7 +357,10 @@ func (h *HelmHandler) Tests(c *gin.Context) {
 		return
 	}
 
-	out, err := h.Data.RunTests(qp.Namespace, qp.Name)
+	panic("Needs implementing")
```

Comment:
@undera 
Why was this changed?
what is expected there?

Reply:
The whole `data.go` file is gone, the approach to interact with Helm has changed into object model. 
You need to implement the new way of test functionality. It would involve `action.NewReleaseTesting(cfg)` from Helm codebase

Reply:
Just look at other handlers, you'll get the idea

Reply:
Created PR #191